data/Bishwajit-2810.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Bishwajit Chakraborty",
+  "bio": "Open Source Enthusiest | A **Learner** | Community ",
+  "tags": [
+    "Open Source",
+    "C++",
+    "Java",
+    "Dart",
+    "Flutter",
+    "DevOps"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/Bishwajit_2810" },
+    { "icon": "FaGithub", "url": "https://github.com/Bishwajit-2810" },
```

Comment:
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/Bishwajit-2810" }
```

Reply:
I don't think `,` at the end is required

Reply:
Ok